src/CodeGenerator/Assembler/NullableGetterAssembler.php

```diff
@@ -59,7 +59,7 @@ public function assemble(ContextInterface $context)
                         'tags' => [
                             [
                                 'name' => 'return',
-                                'description' => $property->getType(),
+                                'description' => $property->getType() . '|null',
```

Comment:
This should result into a new generated classes too. We should commit that as part of this PR.

Reply:
It's done locally, it's just that as this task was not in the sprint, I didn't dedicate time on it for now.

Reply:
Fixed.